packages/admin/tests/test-helper.ts

```diff
@@ -1,12 +1,15 @@
-// @ts-ignore
-import Application from '../app';
-import config from '../config/environment';
+import Application from 'dummy/app';
```

Comment:
```suggestion
// @ts-ignore
import Application from 'dummy/app';
```

Reply:
Missed that this was in admin, I'm guessing the reason both are failing is because you should move these changes to `data/tests/test-helper.ts` and revert this file